source/write/bulk-write.txt

```diff
@@ -0,0 +1,287 @@
+
+``DeleteOne`` removes *the first* document that matches your query filter.
+``DeleteMany`` removes *all* documents that match your query filter.
+
+The following example creates an instance of ``DeleteOne``:
+
+.. literalinclude:: /includes/write/bulk-write.py
+   :start-after: start-bulk-delete-one
+   :end-before: end-bulk-delete-one
+   :language: python
+   :copyable:
+
+The following example creates an instance of ``DeleteMany``:
+
+.. literalinclude:: /includes/write/bulk-write.py
+   :start-after: start-bulk-delete-many
+   :end-before: end-bulk-delete-many
+   :language: python
+   :copyable:
+
+Mixed Bulk Write Operations
+----------------------------
+
+You can perform multiple types of write operations in a single ``bulk_write()``
+method call. To do so, pass a list of operation class instances to the
+``bulk_write()`` method. By default, the method runs each operation in the order
+it's provided.
+
+The following example performs multiple write operations by using the
+``bulk_write()`` method:
+
+.. literalinclude:: /includes/write/bulk-write.py
+   :start-after: start-bulk-write-mixed
+   :end-before: end-bulk-write-mixed
+   :language: python
+   :copyable:
+
+If any of the write operations fail, {+driver-short+} raises a
+``BulkWriteError`` and does not perform any further operations.
+``BulkWriteError`` provides a ``details`` attribute that includes the operation
+that failed, and details about the exception.
+
+.. note::
+
+   When PyMongo runs a bulk operation, it uses the ``write_concern`` of the
+   collection in which the operation is running. The driver reports all write
+   concern errors after attempting all operations, regardless of execution order. 
+
+Customize Bulk Write Operations
+-------------------------------
+
+The ``bulk_write()`` method optionally accepts additional
+parameters, which represent options you can use to configure the bulk write
+operation. If you don't specify any additional options, the driver does not customize
+the bulk write operation.
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``ordered``
+     - | If ``True``, the driver performs the write operations in the order
+         provided. If an error occurs, the remaining operations are not
+         attempted. If ``False``, the driver performs the operations in an
+         arbitrary order and attempts to perform each operation.
+       | Defaults to ``True``.
+
+   * - ``bypass_document_validation``
+     - | Specifies whether the operation bypasses document-level validation. For more
+         information, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``False``.
+
+   * - ``session``
+     - | An instance of ``ClientSession``. For more information, see the `API
+         documentation <{+api-root+}pymongo/client_session.html#pymongo.client_session.ClientSession>`__.
```

Comment:
Could put this link under API Documentation at page end and leave it out here

Reply:
I decided to put this here instead on this page so that the API documentation section would have links to just things that are directly `bulk_write()` related. I guess this technically is related to it, but its just a general option so not really bulk write specific